File: main/java/com/smart/alerts/models/GeoFenceRequest.java
package com.smart.alerts.models;

public class GeoFenceRequest {
    public long userId;
    public double userLat;
    public double userLon;
    public String deviceToken;

    public GeoFenceRequest() {
        this.userId = 0;
        this.userLat = 0;
        this.userLon = 0;
        this.deviceToken = "";
    }
}

File: IOModels/GeoFenceRequest.cs
namespace SmartAlerts.API.IOModels
{
    public class GeoFenceRequest
    {
        public int UserId { get; set; }         // Customer ID
        public double UserLat { get; set; }
        public double UserLon { get; set; }
        public string DeviceToken { get; set; }
    }

}
